db/migrations/20201028150700_podcasts.js

```diff
@@ -3,17 +3,18 @@ exports.up = function (knex) {
     return knex.schema
       .createTable('podcasts', function (table) {
         table.increments('id').primary();
-        table.string('podcast_id');
```

Comment:
Nice choice getting the podcast id out of here.  We could just refer to the id given by the table.

Reply:
You are right I think this column was not needed for the scope of this project.